compiler/src/iree/compiler/Modules/IO/Parameters/Conversion/ParamsToVM/test/parameter_ops.mlir

```diff
@@ -0,0 +1,134 @@
+// RUN: iree-opt --iree-vm-target-index-bits=64 --split-input-file \
+// RUN:   --iree-vm-conversion --mlir-print-local-scope %s | FileCheck %s
+
+// CHECK-LABEL: @parameterLoad
+// CHECK-SAME: (%[[DEVICE:.+]]: !vm.ref<!hal.device>, %[[QUEUE_AFFINITY:.+]]: i64, %[[WAIT:.+]]: !vm.ref<!hal.fence>, %[[SIGNAL:.+]]: !vm.ref<!hal.fence>)
+func.func @parameterLoad(%device: !hal.device, %queue_affinity: i64, %wait: !hal.fence, %signal: !hal.fence) -> !hal.buffer {
+  %c50_i64 = arith.constant 50 : i64
+  %c100 = arith.constant 100 : index
+  // CHECK-DAG: %[[SCOPE:.+]] = vm.rodata.inline {{.+}} = "scope"
+  // CHECK-DAG: %[[KEY:.+]] = vm.rodata.inline {{.+}} = "key"
+  // CHECK: %[[TARGET_BUFFER:.+]] = vm.call @io_parameters.load
+  // CHECK-SAME: (%[[DEVICE]], %[[QUEUE_AFFINITY]], %[[WAIT]], %[[SIGNAL]],
+  // CHECK-SAME:  %[[SCOPE]], %[[KEY]], %c50, %c48, %c527363, %c100)
+  %target_buffer = io_parameters.load<%device : !hal.device> affinity(%queue_affinity) wait(%wait) signal(%signal) source("scope"::"key")[%c50_i64] type("DeviceVisible|DeviceLocal") usage("TransferSource|TransferTarget|Transfer|DispatchStorageRead|DispatchStorageWrite|DispatchStorage|SharingImmutable") : !hal.buffer{%c100}
```

Comment:
Would line breaks make sense in this op printer? This line is very long

Maybe like this? (Can't do much about the usage enums besides add collections of enums...?)

```mlir
%target_buffer = io_parameters.load<%device : !hal.device> {
  affinity(%queue_affinity)
  wait(%wait)
  signal(%signal)
  source("scope"::"key")[%c50_i64]
  type("DeviceVisible|DeviceLocal")
  usage("TransferSource|TransferTarget|Transfer|DispatchStorageRead|DispatchStorageWrite|DispatchStorage|SharingImmutable")
} : !hal.buffer{%c100}
```

(similar question for the other `io_parameters` ops)

Reply:
at the point in the IR where these appear there's lots of long stuff, so it's fine - most investigation will happen at the stream level where things are nicely formatted/terse (these are just alongside all the other hal ops that look the same)